Desert puzzle implementation

Nicovi15 opened the pull request:
Les différentes interactions des ateliers sont fonctionnelles. 
Je pense qu'il faudra juste revoir la taille de certains éléments et plus globalement la distance entre chaque atelier parce qu'en jeu je trouve ça un peu trop grand.
Il faudra aussi : 
- placer correctement les splines des pipes une fois qu'on aura un placement définitif
- remplacer les place holders
- ajouter des sfx aux éléments le nécessitant

Nicovi15 committed:
Implement desert biome interactions

Nicovi15 committed:
Add pipes

Nicovi15 commented:
Je viens aussi de voir que j'ai commit BaseMap, mais c'est une erreur d'inattention. Lors du merge, on pourra écraser ma modification.

Lisouille69 committed:
Rapprochement des ateliers du désert + réduction des assets de l'atelier grotte

WIP

Lisouille69 committed:
modification de la position des tuyaux

Lisouille69 committed:
Positions tuyaux + suppression d'une vanne et d'une vanne transparente en trop

HugoPasquier committed:
Remove clone of BP_ValveMissing in BaseMap

HugoPasquier committed:
Unlarge Sound Volume to all desert

HugoPasquier committed:
Adapt Desert & Cave sound volume to avoid overlap

HugoPasquier committed:
Activate Shadow sound on cave enter to avoid sound through desert floor

HugoPasquier committed:
Add Valve Pick Up, Placement and Rotation sound

HugoPasquier committed:
Add Completion Sound for each pipe activation

HugoPasquier committed:
Remplace valve cylinder by part of the alambic

HugoPasquier committed:
Adapt Valve position and makein face cam when picked up

HugoPasquier committed:
Remove white wall in front of lighthouse

HugoPasquier committed:
Merge remote-tracking branch 'origin/dev_v0.X' into DesertImplementation

# Conflicts:
#	Content/Levels/BaseMap.umap
#	Content/Levels/SubLevels/Desert/DesertTemp.umap

HugoPasquier committed:
Add fade to propper loop in ambiance desert sound

adrimyla committed:
Merge branch 'dev_v0.X' into DesertImplementation

adrimyla reviewed (commented):
C'est top !! Ca marche vraiment bien 😁 👍
Pour moi il y a qu'une petite chose c'est qu'il faudrait peut être mettre l'eau "à plat" car le je trouve que c'est bizarre que ça soit penché comme ça
![image](https://github.com/SL-GAME/SansLogiqueGame/assets/57765484/6440f19d-5b81-494c-b1c5-f8842e0149a0)

HugoPasquier committed:
Merge remote-tracking branch 'origin/dev_v0.X' into DesertImplementation

# Conflicts:
#	Content/FMOD/BP_SoundManager.uasset
#	Content/FMOD/Desktop/Master.strings.bank
#	Content/FMOD/Desktop/SoundDesign.bank
#	Content/FMOD/PrivateIntegrationData/AssetLookup.uasset

HugoPasquier committed:
Fix Fmod merging problems

Lisouille69 committed:
Modification oasis

- redressement du plane de l'eau
- modification du landscape sous l'eau
- modification du chemin des rochers
